### ISSUE TITLE
Sign the installers

### ISSUE THREAD

**Issue by jeroen** [rwinlib/base#6]
Figure out how to sign the R installers on appveyor. @dlemstra seems to have already [figured this out ](https://github.com/ImageMagick/ImageMagick-Windows/blob/master/appveyor.yml) for imagemagick.

**Comment by dlemstra** [comment]
Feel free to ping me if you need help with this.

**Comment by jeroen** [comment]
I assume I need to purchase a cert first? What CA do you recommend? Or does windows accept self signed certs?

**Comment by dlemstra** [comment]
I think you could use a self signed certificate but then your installer will still not show up as trusted. We use a certificate from leaderSSL that was donated to us. I think you do need to "be a organization" to get a certificate from then.

**Comment by jeroen** [comment]
Fixed via https://github.com/rwinlib/base/commit/c0c522bfc3615d284f4d6c086810da72eac7c238, mostly copied from @dlemstra setup. Thanks!

@dlemstra do you know if the "timestamp" server has to match the brand of the cert I purchased, or can it be any timestamp server?

**Comment by dlemstra** [comment]
It can be any timestamp server, this one worked the best for us.

**Comment by jeroen** [comment]
Hmm I just tested and my installer is signed, however I still get the windows defender warning. 

<img width="524" alt="screen shot 2018-03-20 at 4 52 42 pm" src="https://user-images.githubusercontent.com/216319/37666161-2807e5ca-2c5f-11e8-956b-f545b3aec3c9.png">

Is that something that will disappear eventually? Or should I have gotten a more expensive cert (I used comodo via leaderSSL)

**Comment by dlemstra** [comment]
I don't think this will ever disappear. I am not sure but it might be possible that the More info shows your certificate.

**Comment by jeroen** [comment]
Hmm I just checked one of your weekly imagemagick installers and there was no warning. I think I just have to build up some certificate reputation via legitimate downloads.

Are you using a regular or EV-cert?

**Comment by dlemstra** [comment]
We have a regular certificate.

**Comment by jeroen** [comment]
This seems to work. Even though my cert is still doesn't have enough reputation :(

**Comment by jeroen** [comment]
Still getting the Windows defender warnings. Maybe we also need to run the [Windows App Certification Kit](https://msdn.microsoft.com/en-us/library/windows/desktop/mt637086.aspx?f=255&MSPPError=-2147217396). According to [this page](https://msdn.microsoft.com/en-us/library/windows/desktop/mt637082.aspx) we can simply run the following line on the installer:

```
appcert.exe reset
appcert test -apptype desktop -setuppath d:\cdrom\setup.exe -appusage peruser -reportoutputpath [report file name]
```
